lib/Auth/Process/Statistics.php

```diff
@@ -27,11 +28,16 @@ public function __construct($config, $reserved)
 
     public function process(&$request)
     {
+        if (empty($this->config->getString('userIdAttribute'))) {
+            $this->userIdAttribute = $request['rciamAttributes']['cuid'];
+        } else {
+            $this->userIdAttribute = $request['Attributes'][$this->config->getString('userIdAttribute')];
+        }
```

Comment:
if rciamAttributes-->cuid is not available or the configured userIdAttribute is not available then log.error and return without throwing an exception

Reply:
Done